_includes/demos/buy-contract-python.py

```diff
@@ -1,35 +1,32 @@
-
-# This is an example of using Python to buy a contract via the Deriv/Binary API
-# This is a simple example of buying a contract where we do not check first if
-# the user has the symbol available to buy. To check this you would use the active_symbols call.
+# This is an example of using Python to buy a contract via the Deriv/Binary API.
+# This is a simple example of buying a contract where we do not check first ifthe user has the
```

Comment:
Forgot the space in "ifthe" - `check first if the user`

Reply:
Eagle, eye. Replaced.